<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MFigure</class>
 <widget class="QWidget" name="MFigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MFigure</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QCustomPlot" name="plot_lnglat" native="true"/>
   </item>
   <item row="2" column="2">
    <widget class="QCustomPlot" name="plot_progress" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QCustomPlot" name="plot_xy" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QCustomPlot" name="plot_rudder" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QCustomPlot" name="plot_heel" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QCustomPlot" name="plot_thrust" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QCustomPlot" name="plot_pitch" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="QCustomPlot" name="plot_velocity" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="QCustomPlot" name="plot_yaw" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
